site/docs/samples/dashboard-sample.md

```diff
@@ -76,47 +72,46 @@ helm install rel1-backend-server ./tmp/backend_server
+In this example we use three of the endpoints that the backend data server exposes. For each endpoint, we define 
+a Fybrik *Asset* describing the data that will be returned as a response from the backend data server. In particular, the description of the Asset characterizes the type of data returned by that endpoint, as well as describes which fields within the data contain sensitive data.
+This description will be used when policies are applied to the data.  
+
 Register the assets:
 ```bash
 kubectl apply -f https://raw.githubusercontent.com/fybrik/fogProtect-dashboard-sample/main/assets/asset_get_safety_data.yaml
 kubectl apply -f https://raw.githubusercontent.com/fybrik/fogProtect-dashboard-sample/main/assets/asset_start_robot.yaml
 kubectl apply -f https://raw.githubusercontent.com/fybrik/fogProtect-dashboard-sample/main/assets/asset_stop_robot.yaml
 ```  
-The identifier of the assets is `<namespace>/<name>`, and it should be used in the `FybrikApplication` that we will 
-describe later on. For example the identifier of the asset defined in the file `asset_start_robot.yaml` is 
+The identifier (name) of the asset reflects the URL of the end point, and is of the form: `<namespace>/<name>`, where backslash characters ("/") in the URL are replaced by a period (".") in the asset name.
+For example, the name of the asset in file asset_start_robot.yaml, represented by the URL, <HOST>/api/control/start-robot, is 
 `fogprotect/api.control.start-robot`.  
 
-## Create JWT authentication key
+## Create the JWT authentication key
 
-As the HTTP requests should contain the role of the user in the header, the dashboard application uses JWT to pass 
-the JWT of the relevant role in the header. The JWT is authenticated using a secret key that we store as a secret 
+As the HTTP requests from the dashboard should contain the role of the logged in user in the header, the dashboard application uses JWT to encode
+the relevant role in the header. The JWT is authenticated using a secret key that we store as a secret 
 in the cluster, in both `fogprotect` and `fybrik-blueprints` namespaces.  
 Create the JWT secret:  
 ```bash
 kubectl apply -f https://raw.githubusercontent.com/fybrik/fogProtect-dashboard-sample/main/secrets/jwt_key_secret.yaml
 kubectl apply -n fybrik-blueprints -f https://raw.githubusercontent.com/fybrik/fogProtect-dashboard-sample/main/secrets/jwt_key_secret.yaml
 ```
 
-## Fybrik manager RBAC
+## Fybrik Manager RBAC
 
-In order for the fybrik manager to be able to access the assets and to deploy the fybrik module that we created, give 
-the manager the relevant [RBAC authorization](https://github.com/fybrik/fogProtect-dashboard-sample/blob/main/fybrik-system-manager-rbac.yaml):  
+In order for the Fybrik Control Plane to be able to access the JWT key that we created, give 
+the Manager the relevant [RBAC authorization](https://github.com/fybrik/fogProtect-dashboard-sample/blob/main/fybrik-system-manager-rbac.yaml):  
```

Comment:
why do we need this RBAC and access to jwt key?
More generally about the sample, it has cluster roles --- which it shouldn't. Please reduce the RBAC that the sample needs.

Reply:
I don't know - Robert added this code.  I need to investigate

Reply:
I will fix this code.  A secret key is required for JWT creation and verification.  The GUI will require the key for JWT creation and the module for verification.  The current code creates two copies of this key, one in the fogprotect ns (for the GUI) and the other in the fybrik-blueprint ns for the module.  The code uses the serviceaccount for the Manager, and then due to ns issues, creates a global binding.  This of course is not the way to go.

Instead, the code should be fixed as follows:
1. The module creates a serviceaccount to handle the JWT secret.  This will be created in the fybrik-system ns
2. A single secret is created for the JWT
3. Define a ClusterRole for a jwt-secret-reader-role which only gives verbs for reading (get, list...)
4. Define a RoleBinding associated with the JWT secret in both the fybrik-blueprint ns and in the fogprotect ns to give the GUI and module access to the secret.  Note that the RoleBinding restricts the ClusterRole definitions to the specified namespace.